### PR TITLE
ESD-473 : API Review

### PR DESCRIPTION
**Pull request by jehovahmidara**

### CODE REVIEW

File: MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/guidance/GuidanceManeuverListener.java [hunk 1]
```diff
@@ -27,8 +27,8 @@
      * Called when there is a change in {@link GuidanceManeuverData}.
      *
      * @param data
-     *         The {@link GuidanceManeuverData} containing the latest maneuver data. This can be
-     *         null in case there is no data available - for example, when the GPS signal is lost
+     *         The {@link GuidanceManeuverData} containing the latest maneuver data. In case there
```
Comment (datasun) [inline MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/guidance/GuidanceManeuverListener.java]: Please revert this change, this is a callback.

File: MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/guidance/base/BaseGuidancePresenter.java [hunk 2]
```diff
@@ -204,14 +204,14 @@ protected void handleGpsRestore() {
...
-     * Notifies when route reroute begin.
+     * Notifies when rerouting begins.
      */
     protected void handleRerouteBegin() {
         // Let the sub class override it, if interested.
     }
 
     /**
-     * Notifies when route reroute end with success.
+     * Notifies when route reroute ends with success.
```
Comment (datasun) [inline MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/guidance/base/BaseGuidancePresenter.java]: Same here, please.

File: MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/guidance/base/BaseGuidancePresenter.java [hunk 3]
```diff
@@ -221,14 +221,14 @@ protected void handleRerouteEnd(RouteResult routeResult) {
     }
 
     /**
-     * Notifies when route reroute end with failure.
+     * Notifies when route reroute ends with failure.
```
Comment (datasun) [inline MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/guidance/base/BaseGuidancePresenter.java]: Same here, please.

File: MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/guidance/base/BaseGuidancePresenter.java (hunk 2 above)
Comment (jehovahmidara) [reply MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/guidance/base/BaseGuidancePresenter.java]: Done.

File: MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/guidance/base/BaseGuidancePresenter.java (hunk 3 above)
Comment (jehovahmidara) [reply MSDKUIKit/MSDKUILib/src/main/java/com/here/msdkui/guidance/base/BaseGuidancePresenter.java]: Done
